jobs/exportCovid19.ts

```diff
@@ -0,0 +1,219 @@
+import * as moment from 'moment';
+import { Prestacion } from '../modules/rup/schemas/prestacion';
+import { InformacionExportada } from '../core/log/schemas/logExportaInformacion';
+import { handleHttpRequest } from '../utils/requestHandler';
+import { sisa } from './../config.private';
+const fs = require('fs');
+
+const user = sisa.username;
+const clave = sisa.password;
+const urlSisa = sisa.url_nomivac;
+
+export async function exportCovid19(done, horas) {
+    const start = moment(new Date()).subtract(horas, 'h').format('YYYY-MM-DD HH:mm:ss');
+    const end = moment(new Date()).format('YYYY-MM-DD HH:mm:ss');
+    const pipelineVacunaCovid19 = [
+        {
+            $match: {
+                'solicitud.fecha': {
+                    $gte: new Date(start), $lte: new Date(end)
+                },
+                'estadoActual.tipo': 'validada',
+                'ejecucion.registros.concepto.conceptId': '840534001'
+
+            }
+        },
+        {
+            $sort: {
+                'solicitud.fecha': 1
+            }
+        },
+        {
+            $project: {
+                paciente: {
+                    idPaciente: '$paciente.id',
+                    dni: '$paciente.documento',
+                    nombre: '$paciente.nombre',
+                    apellido: '$paciente.apellido',
+                    sexo: '$paciente.sexo',
+                    fechaNacimiento: '$paciente.fechaNacimiento'
+                },
+                fecha: '$solicitud.fecha',
+                idEfector: '$solicitud.organizacion.id',
+                vacuna: '$ejecucion.registros.valor.vacuna'
+            }
+        },
+
+        { $project: {
+            _id: 0,
+            idPaciente: '$paciente.idPaciente',
+            dni: '$paciente.dni',
+            sexo: '$paciente.sexo',
+            nombre: '$paciente.nombre',
+            apellido: '$paciente.apellido',
+            fechaNacimiento: '$paciente.fechaNacimiento',
+            fecha: '$fecha',
+            idEfector: '$idEfector',
+            vacuna: '$vacuna'
+        }
+        },
+        {
+            $lookup: {
+                from: 'organizacion',
+                localField: 'idEfector',
+                foreignField: '_id',
+                as: 'organizacion'
+            }
+        },
+        {
+            $addFields: {
+                organizacion: {
+                    $arrayElemAt: [
+                        '$organizacion',
+                        0
+                    ]
+                }
+            }
+        },
```

Comment:
```suggestion
        { $unwind: '$organizacion'}
```

Mongo aplica optimizaciones cuando hay un lookup seguido de un unwind.